docs/versioned_docs/version-0.14.0/general/02-Components/AskUI-Development-Environment.md

```diff
@@ -209,16 +209,11 @@ The ADE is build to work seamlessly with (VSCode)[https://code.visualstudio.com/
     "livePreview.customExternalBrowser": "Default",
     "terminal.integrated.profiles.windows":{   
         "askui-shell": {    
-            "path": ["C:\\Program Files\\AskUI GmbH\\AskUI Suite\\Tools\\askui-shell.cmd"],
-            "args": [
-               "activate"
-            ],
-            "icon": "terminal-bash",
-        },        
-        "Git Bash": null,        
-        "Command Prompt": null,        
-        "PowerShell": null,        
-        "JavaScript Debug Terminal": null
```

Comment:
When you enable the shells, than you should change the `terminal.integrated.profiles.windows` explaination.